src/CglLandP/CglLandPSimplex.hpp

```diff
@@ -298,7 +298,7 @@ class CglLandPSimplex
     TabRow original_row_k_;
     /** Row of leaving candidate*/
     TabRow row_i_;
-#ifndef NDBEUG
+#ifndef NDEBUG
     TabRow new_row_;
 #endif
```

Comment:
It's usually not a good idea to have the definition of a public struct depend on whether `NDEBUG` is defined or not, since difficult to debug issues can appear if Cgl and the code that builds against Cgl have a different understanding on what is where in the struct.

Reply:
I'll defer to John on that, but I guess there's no reason not to just include it all the time.